docs/cli.md

```diff
@@ -468,22 +469,22 @@ gcli tag details --tag tagA
 gcli tag list
 ```
 
-#### Delete a tag
+#### Delete tags
 
 ```bash
-gcli tag delete --tag tagA
+gcli tag delete --tag tagA tagB
 ```
 
-#### Add a tag to an entity
+#### Add tags to an entity
 
 ```bash
-gcli tag set --name catalog_postgres.hr --tag tagA
+gcli tag set --name catalog_postgres.hr --tag tagA tagB
```

Comment:
This is confusing too.

The `--tag` option for the `tag`/`tags` command is weird.

```suggestion
gcli tags set --name catalog_postgres.hr tagA tagB
```

Or a different approach, where we don't separate tags with spaces in command line.

```suggestion
gcli set --name catalog_postgres.hr --tags tagA,tagB
```

Or yet another valid syntax, where the `--tag` option can be specified multiple times.

```suggestion
gcli set --name catalog_postgres.hr --tag tagA --tag tagB
```

Reply:
> This is confusing too.
> 
> The `--tag` option for the `tag`/`tags` command is weird.
> 
> Or a different approach, where we don't separate tags with spaces in command line.
> 
> Or yet another valid syntax, where the `--tag` option can be specified multiple times.

'tag'  is command entity,  '—tag' is a parameter